sources/packages/backend/apps/db-migrations/src/sql/CRAIncomeVerifications/Standardize-monetary-columns.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE
+    sims.cra_income_verifications
+alter column
+    reported_income type NUMERIC(8, 2);
```

Comment:
The Student application right now accepts a max income of up to 100000000 (defined by business).
When the workflow tries to save a value like that into the `cra_income_verifications` it will generate an error.
We should either allow this column to receive a `NUMERIC(11, 2)` or reduce the constraint in the UI to allow up to 999,999.

Reply:
Besides that, we currently keep a const `MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE` to apply some sort of max value validation when there is no business direction on it.
This const is also currently defined as 100,000,000 which would cause an issue because the `NUMERIC(8, 2)` would not support it (issue already in place).

Reply:
updated

Reply:
The same applies to offerings. Let's settle the conversation in the DEVs chat for a final direction.

Reply:
@katrina-aot-git I am trying to get feedback from the DEV but I feeling from the chat right now would be leaning towards reverting the changes for offerings an CRA income.
Please let me know if there is a concern or if we should have a meeting with the DEVs to settle the conversation.
@dheepak-aot @guru-aot @lewischen-aot @bidyashish @sh16011993 please share you thoughts in favor or against it.